src/main/webapp/app/overview/exercise-details/course-exercise-details.component.html

```diff
@@ -18,7 +18,7 @@
             [showInOrionWindow]="false"
             [courseId]="courseId"
             [exercise]="exercise"
-            [showResult]="showResults && currentResult !== null"
+            [showResult]="showResults && latestRatedResult !== undefined"
```

Comment:
```suggestion
            [showResult]="showResults && latestRatedResult != undefined"
```
Then you check for both `undefined` and `null` :)